Agent - allow the user to set the agent address - ManagerAddress (#204)
engelmi/hirte: doc/man/hirte-agent.1.md

The port on which `hirte` is listening for connection request and the `hirte-agent` is connecting to. This option will overwrite the port defined in the configuration file.

#### **--address**, **-a**

> SD Bus address used by `hirte-agent` to connect to `hirte`. See `man sd_bus_set_address` for its format.

Reviewer: Just call it "dbus address". This is a standard format used by all dbus libraries, specified in the dbus spec.

Reviewer: reference: 
https://dbus.freedesktop.org/doc/dbus-specification.html#addresses

Author: Fixed

> Overrides any setting of `ManagerHost` or `ManagerPort`.
> This option will overwrite the port defined in the configuration file.

Reviewer 2: ```suggestion
Overrides any setting of `ManagerHost` or `ManagerPort` defined in the configuration file as well as the respective CLI options. 
```

Author: Done. I've changed it in both files (since the address takes precedence over any host:port configuration)
